genotype_api/api/endpoints/samples.py

```diff
@@ -1,33 +1,34 @@
 from typing import List, Optional, Literal
 from fastapi import APIRouter, Depends, Query
 from fastapi.responses import JSONResponse
-from datetime import datetime, timedelta, date
+from datetime import timedelta, date
 from starlette import status
 
+import genotype_api.database.crud.create
 from genotype_api.constants import SEXES
 from genotype_api.database import get_session
 from genotype_api.match import check_sample
 from genotype_api.models import (
-    Sample,
-    SampleReadWithAnalysis,
-    SampleRead,
-    User,
     SampleDetail,
-    Analysis,
     MatchResult,
     MatchCounts,
+)
+from genotype_api.database.models.models import (
```

Comment:
```suggestion
from genotype_api.database.models import (
```
Or do we need the extra level?